### ISSUE TITLE
port to Hyprland

### ISSUE THREAD

**Issue by ildar** [artemsen/swaykbdd#12]
does swaykbdd work with Hyprland? or any other app for Hyprland?

**Comment by artemsen** [comment]
No, `swaykbdd` doesn't support Hyperland.
The [Sway IPC](https://man.archlinux.org/man/sway-ipc.7.en) is not compatible with [Hyperland's one](https://wiki.hyprland.org/IPC).

**Comment by ghost** [comment]
what about i3?

P.S. if you replace SWAYSOCK with I3SOCK [here](https://github.com/artemsen/swaykbdd/blob/master/src/sway.c#L143) it will work in both i3 and sway, though per tab switching won't work in i3

**Comment by artemsen** [comment]
Swaykbdd uses Sway IPC to communicate with the WM, and this is the biggest
part of the project. To support Hyprland this part needs to be fully
rewritten, so it will be a HyprlandKbdd :) Sorry, I don't have any plans to
implement this, at least for now.

For X11 sessions (i3) this functionality available in
https://github.com/qnikst/kbdd

**Comment by artemsen** [comment]
> what about i3?

Yeah, it is compatible, but there is no guarantee that this will work.
Your can run the app without modification: 
`SWAYSOCK=$I3SOCK swaykbdd`

For X11 sessions this functionality available in https://github.com/qnikst/kbdd
